src/pcapi/routes/pro/offerers.py

```diff
@@ -110,6 +115,31 @@ def get_offerer(offerer_id: str) -> GetOffererResponseModel:
     return GetOffererResponseModel.from_orm(offerer)
 
 
+@private_api.route("/offerers/<humanized_offerer_id>/stats", methods=["GET"])
+@login_required
+@spectree_serialize(on_success_status=200, response_model=GetOffererResponseModel)
+def get_venues_by_offerer_with_stats(humanized_offerer_id: str) -> GetOffererResponseModel:
+    check_user_has_access_to_offerer(current_user, dehumanize(humanized_offerer_id))
+    offerer = load_or_404(Offerer, humanized_offerer_id)
+
+    venue_stats_by_ids = {}
+    active_bookings_quantity_by_venue = get_active_bookings_quantity_for_offerer(offerer.id)
+    validated_bookings_quantity_by_venue = get_validated_bookings_quantity_for_offerer(offerer.id)
+
+    for managedVenue in offerer.managedVenues:
```

Comment:
(détail / style)

```suggestion
    for venue in offerer.managedVenues:
```

(ou au moins `managed_venue` en snake_case)